newspaper/extractors.py

```diff
@@ -429,19 +429,19 @@ def get_meta_img_url(self, article_url, doc):
         """
         top_meta_image, try_one, try_two, try_three, try_four = [None] * 5
         try_one = self.get_meta_content(doc, 'meta[property="og:image"]')
-        if try_one is None:
-            link_icon_kwargs = {'tag': 'link', 'attr': 'rel', 'value': 'icon'}
-            elems = self.parser.getElementsByTag(doc, **link_icon_kwargs)
-            try_two = elems[0].get('href') if elems else None
-
-        if try_two is None:
+        if try_one == '':
```

Comment:
You are right. When you replace `if try_one is none` with `if not try_one`, the difference is unclear, and i wrote more explicit way.